geometry_msgs/Polygon[] polygons
float32 originX
float32 originY
float32 lenX
float32 lenY
